tests/acceptance/features/apiSharingNg/sharedWithMe.feature

```diff
@@ -806,175 +790,88 @@ Feature: an user gets the resources shared to them
-                            "grantedToV2",
-                            "id",
-                            "invitation",
-                            "roles"
-                          ],
-                          "properties": {
-                            "grantedToV2": {
-                              "type": "object",
-                              "required": [
-                                "group"
-                              ],
-                              "properties":{
-                                "group": {
-                                  "type": "object",
-                                  "required": [
-                                    "displayName",
-                                    "id"
-                                  ],
-                                  "properties": {
-                                    "displayName": {
-                                      "type": "string",
-                                      "enum": ["grp1"]
-                                    },
-                                    "id": {
-                                      "type": "string",
-                                      "pattern": "^%user_id_pattern%$"
-                                    }
-                                  }
-                                }
-                              }
-                            },
-                            "id": {
-                              "type": "string",
-                              "pattern": "^%permissions_id_pattern%$"
-                            },
-                            "invitation": {
+                          "required": ["group"],
+                          "properties":{
+                            "group": {
                               "type": "object",
                               "required": [
-                                "invitedBy"
+                                "displayName",
+                                "id"
                               ],
                               "properties": {
-                                "user":{
-                                  "type": "object",
-                                  "required": [
-                                    "displayName",
-                                    "id"
-                                  ],
-                                  "properties": {
-                                    "displayName": {
-                                      "type": "string",
-                                      "enum": ["Alice Hansen"]
-                                    },
-                                    "id": {
-                                      "type": "string",
-                                      "pattern": "^%user_id_pattern%$"
-                                    }
-                                  }
+                                "displayName": {
+                                  "type": "string",
+                                  "enum": ["grp1"]
```

Comment:
```suggestion
                                  "const": "grp1"
```

Reply:
if you touch then better to change ACC to currently standard

Reply:
They are due to moving scenarios. More refactor in json schema will come later but I have tried to update the lines that the github shows